core/src/test/scala/unit/kafka/network/SocketServerTest.scala

```diff
@@ -1878,6 +1878,93 @@ class SocketServerTest {
     }, false)
   }
 
+  /**
+   * Tests for KAFKA-13559 - Processing request got delayed by 300 ms in the following condition:
```

Comment:
We don't usually add references to tickets in comments. Can we rewrite this comment to say what it is testing and move the explanation of the bug into the PR description that will be included in the commit message?

Reply:
done